.github/workflows/integration-test.yml

```diff
@@ -903,8 +903,8 @@ jobs:
         if: steps.eks-ec2-integration-test.outputs.cache-hit != 'true'
         uses: nick-fields/retry@v2
         with:
-          max_attempts: 3
-          timeout_minutes: 60 # EKS takes about 20 minutes to spin up a cluster and service on the cluster
+          max_attempts: 2
```

Comment:
This test takes a long time to run so we should only retry 2 times so we don't run into timeout issues